modules/ROOT/partials/configuration/paste_data_images.adoc

```diff
@@ -5,11 +5,11 @@ This option specifies whether `+data:url+` images (inline images) should be remo
 
 Setting `+paste_data_images+` to `+true+` will allow the pasted images, while setting it to `+false+` will disallow pasted images.
 
-For example, Firefox enables you to paste images directly into any `+contentEditable+` field. This is normally not something people want, so this option is `+false+` by default. For example, a 600kb embedded image would block page loads and prevents it from being cached on multiple pages.
+For example, Firefox enables you to paste images directly into any `+contentEditable+` field. A 600kb embedded image would block page loads and prevents it from being cached on multiple pages. Setting this option to `+false+` would be prevent such case.
```

Comment:
This paragraph makes no sense... can you please have another go at rewording this or ask for help? We maybe event should just largely remove all this and just mention that it can be set to false to disable the behaviour.

- [ ] Reword this so it makes sense

Reply:
```suggestion
For example, Firefox enables you to paste images directly into any `+contentEditable+` field. This functionality can cause problems for users, such as when attempting to paste an embedded image that is too large. This would block page loads and prevent it from being cached on multiple pages. Setting this option to `+false+` prevents such issues.
```
I made an attempt, but I'm not sure that quite makes sense yet either. 

I do like explaining why this defaults to false, but I definitely think we can generalise the example a bit.

Reply:
@metricjs it doesn't default to `false` though?

Reply:
The `paste_data_image` section reads better without the example paragraph. Beside, it doesn't add much value

Reply:
Agreed, let's remove the example